src/Utilities/Compiler/Extensions/IMethodSymbolExtensions.cs

```diff
@@ -547,7 +547,7 @@ public static bool IsLambdaOrLocalFunction(this IMethodSymbol method)
             switch (method.MethodKind)
             {
                 case MethodKind.LambdaMethod:
-                case MethodKindEx.LocalFunction:
+                case MethodKind.LocalFunction:
```

Comment:
It's a question more than a contribution, but can someone explain why LocalFunction isn't accessible from MethodKind here?
Based on the [docs](https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.methodkind?view=roslyn-dotnet), `MethodKind.LocalFunction` exists. and it was accessible in MethodKindEx, so, why not accessible here?

Reply:
In one of the projects that uses this file, the project intentionally compiles against an old version of Roslyn for maximum compatibility. `MethodKindEx` allows `LocalFunction` to be used if the _runtime_ environment is new enough, without eliminating the ability of older clients to retain full functionality of the analyzer.

Reply:
Thanks!